modules/reference/pages/default-environment-variables.adoc

```diff
@@ -44,7 +45,7 @@ The following table lists the default Open Liberty environment variables and giv
 
 | The user or custom configuration directory that is used to store
   shared and server-specific configuration.
-  See the  `README.TXT` file in the `_path/to/liberty_/wlp` directory for details about shared resource locations.
+  See the `README.TXT` file at the `_path_to_liberty_/wlp/README.TXT` location for details about shared resource locations.
```

Comment:
i think it should be

See the `_path_to_liberty_/wlp/README.TXT` file for details about shared resource locations.